opengemini-client-okhttp/pom.xml

```diff
@@ -22,6 +22,14 @@
             <artifactId>opengemini-client-common</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>io.opengemini</groupId>
+            <artifactId>opengemini-client-test-common</artifactId>
+            <version>${project.version}</version>
+            <classifier>tests</classifier>
+            <type>test-jar</type>
+            <scope>test</scope>
```

Comment:
It is sufficient to set scope to test. Is it necessary to set classfier and type?